test/TokenTimelock.js

```diff
@@ -29,26 +28,26 @@ contract('TokenTimelock', function ([_, owner, beneficiary]) {
   })
 
   it('cannot be released just before time limit', async function () {
-    await increaseTime(moment.duration(0.99, 'year').asSeconds())
+    await increaseTimeTo(this.releaseTime - duration.seconds(3))
```

Comment:
I'm surprised this works. Given the lack of precision of `increaseTimeTo` that we've discussed, I would've thought 3 seconds was not enough to make up for it. Did you try other values?